index.html

```diff
@@ -12,32 +12,41 @@
   <title>My Portfolio</title>
 
   <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-J16KK4QYVH"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-J16KK4QYVH"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+
+    gtag('config', 'G-J16KK4QYVH');
+  </script>
+
+
+</head>
 
-  gtag('config', 'G-J16KK4QYVH');
-</script>
-  
-  
-  </head>
-  
 <body>
   <header>
     <div class="header">
       <h3><a href="#">Welcome</a></h3>
       <nav class="navbar_mobile_ver">
-        <img src="./images/Union.svg" alt="menu icon" />
+        <img src="./images/Union.svg" onclick="mobileMenu()" alt="menu icon" />
       </nav>
     </div>
+    <div class="mobile_menu">
+      <img src="./images/cross_icon.svg" class="cross_icon" onclick="closeMenu()" alt="cross icon">
+      <nav class="navbar_destop_version_alpha">
+        <a onclick="closeMenu()" href="#works">Portfolio</a>
+        <a onclick="closeMenu()" href="#about-me">About</a>
+        <a onclick="closeMenu()" href="#contact-us">contact</a>
+      </nav>
+    </div>
```

Comment:
- [ ] Good job here, kindly make sure the hamburger button and menu follow the design from the chosen template in Figma (e.g., font, colors, images, tests, margins).
- your design
![FireShot Capture 006 - My Portfolio - timmychan99 github io](https://user-images.githubusercontent.com/44624138/145877955-5dd577da-8f28-4589-a9a0-bdd1998eca4e.png)
 - Figma design
![FireShot Capture 005 - Microverse-Student-Project-1 – Figma - www figma com](https://user-images.githubusercontent.com/44624138/145878030-5fcd5e54-7988-4fb3-a8b5-c718f24bd54d.png)

Reply:
I don't get what are you trying to say here. please can clarify ????